### PR TITLE
Add a community software section and move some stuff inside

### PR DESCRIPTION
**Pull request by sebartyr**
## Describe your PR

Add ~~third-party~~ community section in log management page in order to put aside those which are not directly supported by Clever Cloud and developed by community.

## Checklist

- [ ] My PR is related to an opened issue : #
- [ ] I've read the [contributing guidelines](../CONTRIBUTING.md)

## Reviewes
_Who should review these changes?_ @juliamrch @StevenLeRoux

### CODE REVIEW

**Comment by StevenLeRoux** [comment]
Do you mind adding the current Logs API in the supported HTTPS section?

File: content/doc/administrate/log-management.md [hunk 1]
```diff
@@ -228,3 +221,16 @@ clever drain create NewRelicHTTP "https://log-api.eu.newrelic.com/log/v1" --api-
 {{< callout type="warning" >}}
 NewRelic has two zones, **EU** and **US**. An account on one zone is not available on the other, make sure to target the right intake endpoint (`log-api.eu.newrelic.com` or `log-api.newrelic.com`).
 {{< /callout >}}
+
+### Community software
+
+{{< callout type="info">}}
+Community software is not directly supported by Clever Cloud. It is developed by our community. We do not garantee they will be free of problems.
```
Comment (juliamrch) [inline content/doc/administrate/log-management.md:228]:
```suggestion
Community software isn't directly supported by Clever Cloud. It's developed by our community. We don't  guarantee their maintenance or correct functioning.
```

File: content/doc/administrate/log-management.md [hunk 2]
```diff
@@ -228,3 +221,16 @@ clever drain create NewRelicHTTP "https://log-api.eu.newrelic.com/log/v1" --api-
...
+You had better open issues on their Git repositories instead of contacting our support.
```
Comment (juliamrch) [inline content/doc/administrate/log-management.md:229]:
```suggestion
You are better off opening issues on their GitHub repositories than contacting Clever Cloud support.
```

**Review by juliamrch — CHANGES_REQUESTED** [review]
Added some suggestions following Vale checks.

**Comment by sebartyr** [comment]
Hi @juliamrch 
I merged suggestions

**Pull request merged**